sycl/test-e2e/Matrix/element_wise_all_ops.cpp

```diff
@@ -5,20 +5,13 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
-// REQUIRES: matrix
+// REQUIRES: cpu, gpu
+// Due to current bug in A750
```

Comment:
Since we don't know the root cause:
```suggestion
// Test is flaky/timeouts on some variants of DG2 and temporary disabled. Needs to be investigated.
```